### PR TITLE
Changing the templates to use VMSS

### PR DESCRIPTION
**Pull request by ChackDan**
Changing the templates to use VMSS

### CODE REVIEW

File: service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
```
Comment (singhkay) [inline service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json]: Any reason for the move back to the preview schema?

File: service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json [hunk 2]
```diff
@@ -62,25 +62,25 @@
...
       }
     },
     "certificateThumbprint": {
       "type": "string",
             "metadata": {
-        "description": "Certificate Thumbprint"
+      "description": "Certificate Thumbprint"
       }
     },
     "sourceVaultValue": {
```
Comment (singhkay) [inline service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json]: Let's call this keyVaultResourceId to be more clear on the expectation here. sourceVaultValue doesn't really mean anything

File: service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json [hunk 3]
```diff
@@ -241,237 +242,6 @@
     },
     {
       "apiVersion": "2015-05-01-preview",
```
Comment (singhkay) [inline service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json]: apiVersion here can change to 2015-06-15 as well

File: service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json [hunk 4]
```diff
@@ -691,7 +418,176 @@
...
+        "virtualMachineProfile": {
+          "extensionProfile": {
+            "extensions": [
+              {
+                "name": "[concat('ServiceFabricNodeVmExt','_vmNodeType0Name')]",
+                "properties": {
+                  "type": "ServiceFabricNode",
+                  "autoUpgradeMinorVersion": false,
+                  "protectedSettings": {
+                    "StorageAccountKey1": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('supportLogStorageAccountName')),'2015-05-01-preview').key1]",
```
Comment (singhkay) [inline service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json]: apiversion 2015-06-15?

File: service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json [hunk 5]
```diff
@@ -691,7 +418,176 @@
...
+                            },
+                            {
+                              "provider": "Microsoft-ServiceFabric-Services",
+                              "scheduledTransferPeriod": "PT5M",
+                              "DefaultEvents": {
+                                "eventDestination": "ServiceFabricReliableServiceEventTable"
+                              }
+                            }
+                          ],
+                          "EtwManifestProviderConfiguration": [
```
Comment (singhkay) [inline service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json]: How does a user discover the values below?

**Comment by singhkay** [comment]
@ChackDan Any update on this?

**Comment by ChackDan** [comment]
@singhkay - I would like to have these published in time for Build 2016.

**Comment by singhkay** [comment]
@ChackDan Looking at the CI errors, I see itemDisplayName is too long. Can you fix? 

`AssertionError: service-fabric-secure-cluster-5-node-1-nodetype-wad/metadata.json - #/itemDisplayName:Failed "maxLength" criteria`

**Comment by ChackDan** [comment]
I saw this.. Is this to shorten the name of the folder?

Sent from my Phone

On Mar 29, 2016, at 5:24 PM, Kay Singh <notifications@github.com<mailto:notifications@github.com>> wrote:


@ChackDan<https://github.com/ChackDan> Looking at the CI errors, I see itemDisplayName is too long. Can you fix?

AssertionError: service-fabric-secure-cluster-5-node-1-nodetype-wad/metadata.json - #/itemDisplayName:Failed "maxLength" criteria

-

**Comment by singhkay** [comment]
@ChackDan This is for the itemDisplayName property in the metadata file

**Comment by ChackDan** [comment]
I did not make changes to the metadata.json , this time around.. is this a new check you guys have added ?

**Comment by singhkay** [comment]
We probably merged it last time without the fix. Can you fix it this time?

**Comment by ChackDan** [comment]
I am at the airport flying to SFO for build 2016. If possible just file a defect on me and I can fix it next week. We need these templates to be updated by tommorow..

Sent from my Phone

On Mar 29, 2016, at 6:28 PM, Kay Singh <notifications@github.com<mailto:notifications@github.com>> wrote:


We probably merged it last time without the fix. Can you fix it this time?

-

**Comment by singhkay** [comment]
@ChackDan Ah ok! No worries, I'll can merge and fix with a follow up PR.

**Comment by ChackDan** [comment]
Thankyou Kay.

**Pull request merged**